src/utils/print.jl

```diff
@@ -139,6 +139,14 @@ function Base.summary(tech::DeviceParameter)
     return "$(typeof(tech))"
 end
 
+function Base.show(io::IO, ::MIME"text/plain", data::OperationalCost)
```

Comment:
@GabrielKS this probably needs updating

Reply:
Eventually yeah as part of https://github.com/NREL-Sienna/PowerSystems.jl/issues/1048. For this merge though it looks okay:
<img width="644" alt="Screenshot 2024-03-27 at 4 25 49 PM" src="https://github.com/NREL-Sienna/PowerSystems.jl/assets/23368820/8fd4bae5-1a49-4520-8201-977d0bece7a2">